README.md

```diff
@@ -9,8 +9,7 @@ Metrics package.
 
 ### Collected Data
 
-* A unique identifier that is generated by computing the [SHA-1][SHA1] of the
-  machine's [MAC address][MAC].
+* A unique UUID v4 random identifier is generated according to [RFC4122][RFC4122]
```

Comment:
The other entries in the list don't have periods as well. Wouldn't it look rather odd?